templates/sag-cc-builder-repos/README.md

```diff
@@ -25,11 +25,13 @@ Use this template to add public Software AG product and fix repositories to use
 
 Create Empower credentials by running the sag-cc-creds template:
 
+```bash
 sagcc exec templates composite apply sag-cc-creds \
   credentials.username=yourusername \
   credentials.password=yourpassword \
   credentials.key=EMPOWER \
```

Comment:
Elly, it should be empower, because in the next command lowercase is used

Reply:
Actually , it is better to keep EMPOWER  here and change it in the next command to be capital

Reply:
Okay, fixed.